src/definitions/modules/dropdown.js

```diff
@@ -3684,7 +3684,7 @@ $.fn.dropdown = function(parameters) {
                 $currentMenu
                   .transition({
                     animation  : transition + ' out',
-                    duration   : settings.duration,
+                    duration   : settings.transition.hideDuration || settings.duration,
```

Comment:
I think we should document `settings.transition` has higher precedence than `settings.duration`.
Perhaps deprecating `settings.duration` ?

Reply:
Yes, will update the docs accordingly 👍🏼